### PR TITLE
Fix -1781 Password Strength Feature

### LINKED ISSUE #1781: Password Strength Feature
**I'm submitting a ...**
- [x] feature request

**Current behaviour:**

No as such feature.

## Will work on issue ??
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1781 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-  Adding Password strength
ezgif com-video-to-gif 16

### CODE REVIEW

**Comment by sk9331657** [comment]
@ParthS007  @abishekvashok  @gabru-md  @mariobehling  Please review.

**Comment by sk9331657** [comment]
@ParthS007  Please review.

**Comment by sk9331657** [comment]
@yashhere @yashLadha  @gabru-md  Please review this one. 😄

File: frontend/app/templates/components/forms/signup-form.hbs [hunk 1]
```diff
@@ -22,8 +22,12 @@
             <div class="field required">
                 <div class="ui left icon input">
                     <i class="lock icon"></i>
-                    {{input type="password" value=password name="password" placeholder="Password"}}
+                    {{input type="password" value=password  name="password"  id="pass" placeholder="Password"}}
                 </div>
+                <meter  id="password-strength-meter"  value={{level}} ></meter>
+
```
Comment (gabru-md) [inline frontend/app/templates/components/forms/signup-form.hbs]: please remove the extra lines from the file.
then we can merge this into the branch
cheers! thanks!

**Comment by sk9331657** [comment]
gabru-md  Done please merge.

**Comment by sk9331657** [comment]
@ParthS007  Please merge this.

**Comment by sk9331657** [comment]
@ParthS007  @gabru-md  @yashLadha Please merge this.

**Pull request merged**